openapi/paths/storefront/kyc-liveness-sessions.yaml

```
@@ -0,0 +1,81 @@
+post:
+  x-products:
+    - Storefront
+  tags:
+    - KYC documents
+  summary: Create a KYC liveness session
+  operationId: StorefrontPostKycLivenessSession
+  x-sdk-operation-name: create
+  description: Creates a KYC liveness session.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/StorefrontKycLivenessSession.yaml
+        examples:
+          request-liveness-session:
+            summary: Basic liveness session
+            description: |-
+              This example shows how to create a liveness session with a request ID.
+            value:
+              customerId: 'cus_0YV7DDSDD1C8DA64KHH2W33CPF'
+              requestId: 'kyc_req_0YV7JMJ3DBCGRBR7K9D4HVGPP5'
+          no-request-liveness-session:
+            summary: Basic liveness session
+            description: |-
+              This example shows how to create a liveness session without a request ID.
+            value:
+              customerId: '123_456789'
+              requestId: null
+    description: KYC request resource.
+    required: true
+  responses:
+    '201':
+      description: KYC liveness session created.
+      headers:
+        Location:
+          $ref: ../components/headers/Location.yaml
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/StorefrontKycRequest.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '422':
+      $ref: ../components/responses/ValidationError.yaml
+get:
+  x-products:
+    - Storefront
+  tags:
+    - KYC documents
+  summary: Retrieve KYC liveness session
```

suggestion: I think this should be plural
```suggestion
  summary: Retrieve KYC liveness sessions
```